### PR TITLE
Blogging Prompts: add feature description view

### PR DESCRIPTION
**Pull request by ScoutHarris**
Ref: #18176 

This adds a description view to the Blogging Prompts Feature Introduction.
- A bordered stub view for the prompt card, to be populated later.
- The description paragraph.
- The bordered note.

Because we don't have any official way of showing the feature introduction just yet, I have set it to show in `applicationDidBecomeActive` to facilitate testing (by minimizing and relaunching the app). I will leave it for now and change/remove it later. 

To test:
- Enable `bloggingPrompts` feature flag.
- Run the app.
- When the app launches, the Feature Introduction should appear. 
- Verify:
  - A bordered area appears at the top under the header.
  - The `We'll show you...` paragraph is under that.
  - The bordered note is last, with the `Note:` text in bold.

<kbd>bp_fi_content</kbd>

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Feature Introduction/Blogging Prompts/BloggingPromptsFeatureDescriptionView.swift [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import UIKit
+import AuthenticationServices
```
Comment (dvdchr) [inline WordPress/Classes/ViewRelated/Feature Introduction/Blogging Prompts/BloggingPromptsFeatureDescriptionView.swift]: Is `AuthenticationServices` imported by mistake, perhaps?

**Review by dvdchr — APPROVED** [review]
LGTM. :shipit:

One minor issue is that I've noticed the in dark mode, the background color for the `BloggingPromptsFeatureDescriptionView` is kinda different from `FeatureIntroductionViewController`:

Simulator Screen Shot - iPhone 11 Pro Max - 2022-04-02 at 00 26 36


> I have set it to show in applicationDidBecomeActive to facilitate testing (by minimizing and relaunching the app). I will leave it for now and change/remove it later.

👍 !

File: WordPress/Classes/ViewRelated/Feature Introduction/Blogging Prompts/BloggingPromptsFeatureDescriptionView.swift (hunk 1 above)
Comment (ScoutHarris) [reply WordPress/Classes/ViewRelated/Feature Introduction/Blogging Prompts/BloggingPromptsFeatureDescriptionView.swift]: Ah, Xcode being "helpful" again. Thanks, I'll remove it.

**Comment by ScoutHarris** [comment]
> One minor issue is that I've noticed the in dark mode, the background color for the BloggingPromptsFeatureDescriptionView is kinda different from FeatureIntroductionViewController

Thanks! I'll look at it later.

**Pull request merged**